frontend/src/metabase/visualizations/echarts/pie/option/constants.ts

```diff
@@ -0,0 +1,14 @@
+import type { RegisteredSeriesOption } from "echarts";
+
+export const SUNBURST_SERIES_OPTIONS: RegisteredSeriesOption["sunburst"] = {
+  type: "sunburst",
+  sort: undefined,
+  label: {
+    rotate: 0,
+    overflow: "none",
+    lineHeight: 50, // TODO update this later
```

Comment:
Will get the exact values later when polishing, and will confirm it looks good with product+design